WebService/v1/Models/SimulationApiModel/SimulationApiModel.cs

```diff
@@ -188,6 +194,15 @@ public static SimulationApiModel FromServiceModel(Simulation value)
             }
 
```

Comment:
looking at lines 166-168 and the conditions at lines 179, 185, 191, I think lines 166-168 are a bug. If StartTime is MinValue, the API response should not contain the field (and similarly for EndTime and StoppedTime)

Reply:
removed lines 166-168. These values will be set in lines 179, 185, 191